frontend/src/views/CellGuide/components/CellGuideCard/components/MarkerGeneTables/index.tsx

```diff
@@ -191,234 +193,362 @@ export const MARKER_GENES_COMPUTATIONAL_TOOLTIP_TEST_ID =
 export const MARKER_GENES_CANONICAL_TOOLTIP_TEST_ID =
   "marker-genes-canonical-help-tooltip";
 
+function getComputationalMarkerGenesTableRowsAndFilters({
```

Comment:
refactored logic that existed in the main body of the functional component into this function.

Reply:
nit: React useHooks have a convention to prefix with `use`

So maybe rename it to `useComputationalMarkerGenesTableRowsAndFilters()` 🙆‍♂️ ?

here and throughout

Also a quick check, do you see these ESLint errors in VSCode?

<img width="770" alt="Screenshot 2023-08-09 at 8 05 25 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/8fee5a2a-6f9f-43c5-86ae-c34522ae6bb1">
<img width="893" alt="Screenshot 2023-08-09 at 8 05 31 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/159eb8a6-4813-4ef0-81fe-1f91d2e24c95">
<img width="732" alt="Screenshot 2023-08-09 at 8 05 52 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/44cb4411-08d8-4b42-bde2-33cb6312d732">